README-legacy.md

```diff
@@ -420,10 +420,10 @@ ENV KUSTOMIZE_PLUGIN_PATH=$XDG_CONFIG_HOME/kustomize/plugin/
 ARG PKG_NAME=ksops
 
 # Override the default kustomize executable with the Go built version
-COPY --from=ksops-builder /go/bin/kustomize /usr/local/bin/kustomize
+COPY --from=ksops-builder /usr/local/bin/kustomize /usr/local/bin/kustomize
 
 # Copy the plugin to kustomize plugin path
-COPY --from=ksops-builder /go/src/github.com/viaduct-ai/kustomize-sops/*  $KUSTOMIZE_PLUGIN_PATH/viaduct.ai/v1/${PKG_NAME}/
+COPY --from=ksops-builder /usr/local/bin/ksops  $KUSTOMIZE_PLUGIN_PATH/viaduct.ai/v1/${PKG_NAME}/
```

Comment:
Previously I just had `install -D ksops /custom-tools/kustomize-plugins/viaduct.ai/v1/ksops/ksops;`
Should we change the working directory of the container to keep that working?

Reply:
As in change the work dir of the KSOPS image to `/usr/local/bin/`?

Reply:
Yep, all good thanks!